include/inexor/vulkan-renderer/world/collision.hpp

```diff
@@ -0,0 +1,250 @@
+#pragma once
+
+#include <inexor/vulkan-renderer/world/cube.hpp>
```

Comment:
```suggestion
```
`RayCubeCollision` being templated means you don't need this.

Reply:
Done.